network-api/networkapi/templates/fragments/buyersguide/pni_mobile_nav.html

```diff
@@ -1,7 +1,7 @@
 {% load bg_nav_tags localization i18n static wagtailroutablepage_tags %}
 {% get_bg_home_page as home_page %}
 
-<div class="d-md-none mt-0 mb-0 tw-bg-white tw-border-b tw-border-blue-10" id="pni-nav-mobile">
+<div class="d-md-none mt-0 mb-0 tw-bg-white" id="pni-nav-mobile">
```

Comment:
If I am following this correctly, this element only contains the category navigation not the very main navigation. We do you think about updating the `id` to reflect that?

Reply:
Agree the id could be improved. I need to check the codebase to see if there will be any collision though. There are quite a bit of relatively similar id names so I want to be extra careful.

Reply:
@tbrlpld I'm gonna create a new ticket for this. There are too many occurrences of `pni-nav-mobile` in the codebase. (35 results in 9 files). I think this deserve its own ticket.

Reply:
Filed https://github.com/mozilla/foundation.mozilla.org/issues/10088

Reply:
Wow. Thanks for that. I think that is going to be useful.